From 3dbbccd260dbba01e33a043a39d56ccfb0f477ea Mon Sep 17 00:00:00 2001
From: Meng Wei <wei.meng@intel.com>
Date: Mon, 3 Dec 2018 16:52:28 +0800
Subject: [PATCH 1221/1301] media: psys: Enable new FW API for IPU performance

Enable trace event for new FW API used to measure IPU performance.

Change-Id: I85dae5433d48f4dcd251a6f449943b5b2c63d0bc
Signed-off-by: Hongyu Yi <hongyu.yi@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu-fw-psys.c     | 2 ++
 drivers/media/pci/intel/ipu-trace-event.h | 7 ++++---
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/drivers/media/pci/intel/ipu-fw-psys.c b/drivers/media/pci/intel/ipu-fw-psys.c
index 5f0f601..9ffe8ca 100644
--- a/drivers/media/pci/intel/ipu-fw-psys.c
+++ b/drivers/media/pci/intel/ipu-fw-psys.c
@@ -30,6 +30,8 @@ int ipu_fw_psys_pg_processing_cycles(struct ipu_psys_kcmd *kcmd)
 	return 0;
 }
 
+
+
 int ipu_fw_psys_pg_disown(struct ipu_psys_kcmd *kcmd)
 {
 	struct ipu_fw_psys_cmd *psys_cmd;
diff --git a/drivers/media/pci/intel/ipu-trace-event.h b/drivers/media/pci/intel/ipu-trace-event.h
index b5e8d4b..fe61ac2 100644
--- a/drivers/media/pci/intel/ipu-trace-event.h
+++ b/drivers/media/pci/intel/ipu-trace-event.h
@@ -62,8 +62,8 @@ TRACE_EVENT(ipu_pg_kcmd,
 		     unsigned int pg_id, unsigned int load_cycles,
 		     unsigned int init_cycles,
 		     unsigned int processing_cycles),
-	    TP_ARGS(func, id, issue_id, pri, pg_id, load_cycles,
-		    init_cycles, processing_cycles),
+	    TP_ARGS(func, id, issue_id, pri, pg_id, load_cycles, init_cycles,
+		    processing_cycles),
 	    TP_STRUCT__entry(__field(const char *, func)
 			     __field(unsigned int, id)
 			     __field(unsigned long long, issue_id)
@@ -83,7 +83,8 @@ TRACE_EVENT(ipu_pg_kcmd,
 			   __entry->processing_cycles = processing_cycles;),
 	    TP_printk
 	    ("pg-kcmd: func=%s,id=%u,issue_id=0x%llx,pri=%u,pg_id=%d,"
-	     "load_cycles=%u,init_cycles=%u,processing_cycles=%u",
+	     "load_cycles=%u,init_cycles=%u,"
+	     "processing_cycles=%u",
 	     __entry->func, __entry->id, __entry->issue_id, __entry->pri,
 	     __entry->pg_id, __entry->load_cycles, __entry->init_cycles,
 	     __entry->processing_cycles)
-- 
2.7.4

